frontend/src/component/releases/ReleasePlanTemplate/TemplateForm.tsx

```diff
@@ -10,48 +17,99 @@ const StyledInput = styled(Input)(({ theme }) => ({
     marginBottom: theme.spacing(2),
 }));
 
+const StyledForm = styled('form')(() => ({
+    display: 'flex',
+    flexDirection: 'column',
+    height: '100%',
+}));
+
 interface ITemplateForm {
+    mode: 'create' | 'edit';
```

Comment:
Where are we using this?

Reply:
I'll remove as it's not needed for this PR